encoding/varint.ts

```diff
@@ -57,11 +57,41 @@ const U64_VIEW = new BigUint64Array(AB);
  * import { decode } from "https://deno.land/std@$STD_VERSION/encoding/varint.ts";
  *
  * const buf = new Uint8Array([0x8E, 0x02]);
- * decode(buf);
- * // [ 300n, 2 ];
+ * decode(buf); // [ 300n, 2 ];
  * ```
+ *
+ * @deprecated (will be removed in 0.224.0) Use {@linkcode decodeVarint} instead.
```

Comment:
I'd suggest we should remove them in 1.0.0 as `std/encoding` is marked 'Stable' in README.

Reply:
Fixed.